python/heterocl/tvm/schedule.py

```diff
@@ -256,100 +256,178 @@ def reuse_at(self, target, parent, axis, name):
     def partition(self, target, partition_type, dim, factor):
         return _api_internal._SchedulePartition(self, target, dim, factor, partition_type)
 
-    def join(self, target, dst, src, type=_expr.IO.FIFO, depth=1):
-        """ join multiple writes to target tensor """
-        return _api_internal._ScheduleJoin(self, target, src, dst, type, depth)
-
-    def to(self, tensor, dst, src, axis=0,
-           type=_expr.IO.DMA, depth=1, local_buffer=True):
-        """ Stream data to devices or on-chip module
+    def to(self, tensor, dst, src, axis=0, io_type=_expr.IO.DMA, depth=1, burst_len=0):
+        """ Stream data to devices or on-chip module 
```

Comment:
We have `.to` in two different files, which is confusing. In `tvm/schedule.py`, we should call only the C API function. We should move the logic to the HeteroCL `schedule.py`.

Reply:
@Hecmay please fix this.